python/example_load_gen.py

```diff
@@ -201,6 +202,8 @@ def tps(self, context: BasicPythonLoadGenContext, request_count: int) -> int:
 
     @staticmethod
     def percent_requests(context: BasicPythonLoadGenContext, count: int) -> float:
+        # multiply the ration by 100 to get a percentage.  Then multiply by 10, round, and divide by 10 to retain
+        # precision of one digit after the decimal.
         return round((count / context.global_request_count) * 100 * 10) / 10
```

Comment:
`round(..., ndigits=1)` accomplishes this:

```python
In [1]: round(13.4567, 1)
Out[1]: 13.5
```

Reply:
okay gtk.  i ported this from the js code which doesn't have that.

Reply:
fixed in c1e55b86ceaed8ca5827f2c637260931b1d95e68